go/common/log/log.go

```diff
@@ -8,15 +8,15 @@ import (
 
 // These are the keys of the log entries
 const (
-	ErrKey       = "err"
-	SubIDKey     = "subId"
-	CfgKey       = "cfg"
-	TxKey        = "tx"
-	CmpKey       = "cmp"
-	NodeIDKey    = "node_id"
-	NetworkIDKey = "netw_id"
-	BlockHeight  = "block_height"
-	BlockHash    = "block_hash"
+	ErrKey         = "err"
+	SubIDKey       = "subId"
+	CfgKey         = "cfg"
+	TxKey          = "tx"
+	CmpKey         = "cmp"
+	NodeIDKey      = "node_id"
+	NetworkIDKey   = "netw_id"
+	BlockHeightKey = "block_height"
```

Comment:
looks like the values don't follow a convention either. 
I think my initial thought was to keep them short and camel cased.
I see the last ones are long and hyphenated.

I don't have a strong opinion, but we should stick to a convention.
The reasoning for the short camel cased is that it takes less space on the screen , so you can see more of the log

Reply:
I'd typically go for explicit values. Like `network_id` instead of `netw_id` and so on. But yeah, no strong feeling one way or the other.

Reply:
ok. we'll see how they look in the logs. 
Could you please rename them: subId -> subscription_id

Thanks